tests/integration/sugar/test_transaction.py

```diff
@@ -125,24 +127,19 @@ async def test_get_transaction_from_hash_with_min_max_ledgers(self, client):
             sequence=WALLET.sequence,
         )
 
-        # WHEN we sign locally and autofill the transaction
-        signed_payment_transaction = await safe_sign_and_autofill_transaction(
-            payment_transaction, WALLET, client
-        )
-
-        # AND submit the transaction
-        response = await send_reliable_submission(signed_payment_transaction, client)
-        payment_hash = response.result["hash"]
-        payment_ledger_index = response.result["ledger_index"]
+        # WHEN we sign locally, autofill, and submit the transaction
+        response = await sign_and_reliable_submission_async(payment_transaction, WALLET)
+        payment_hash = response.result["tx_json"]["hash"]
+        payment_ledger_index = response.result["validated_ledger_index"]
 
         # THEN we expect to retrieve this transaction from its hash with
         # min_ledger and max_ledger parameters
         payment = await get_transaction_from_hash(
             payment_hash,
             client,
             False,
-            payment_ledger_index - 500,
-            payment_ledger_index + 500,
+            payment_ledger_index - 5,
+            payment_ledger_index + 5,
```

Comment:
This is because there are fewer ledgers in the standalone node - there aren't 500 ledgers to go back in time